### PR TITLE
Make all strings translatable

### LINKED ISSUE #3842: Make all text translatable
## Feature description

Make all displayed text to displayed to users translatable.

---------------

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Fixes #3842 

## Checklist

- [ ] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Comment by spacedmonkey** [comment]
@googlebot I consent.

**Comment by barklund** [comment]
Is `amp` the right namespace for these translatable texts now that we're moving away from the AMP plugin? Should we perhaps just go ahead and call it `web-stories`?

**Comment by swissspidy** [comment]
I‘ll rename the text domain with the move, so no objections there

File: assets/src/edit-story/app/history/useHistoryReducer.js [hunk 1]
```diff
@@ -53,7 +54,12 @@ const reducer = ( size ) => ( state, { type, payload } ) => {
 			};
 
 		default:
-			throw new Error( `Unknown history reducer action: ${ type }` );
+			const text = sprintf(
+				/* translators: %s: Type of error. */
+				__( 'Unknown history reducer action: %s', 'amp' ),
```
Comment (swissspidy) [inline assets/src/edit-story/app/history/useHistoryReducer.js]: This is a tricky one. I'd argue that these kind of errors are more developer facing and should not be translatable. I could not even properly translate this anyway.

File: assets/src/edit-story/panels/index.js [hunk 2]
```diff
@@ -67,7 +71,11 @@ export function getPanels( elements ) {
 				case COLOR: return { type, Panel: ColorPanel };
 				case FONT: return { type, Panel: FontPanel };
 				case TEXT: return { type, Panel: TextPanel };
-				default: throw new Error( `Unknown panel: ${ type }` );
+				default: throw new Error( sprintf(
+					/* translators: %s: Type of panel. */
+					__( 'Unknown panel: %s', 'amp' ),
```
Comment (swissspidy) [inline assets/src/edit-story/panels/index.js]: Same here about developer-facing messaging. I'd not make that translatable

File: assets/src/edit-story/panels/position.js [hunk 3]
```diff
@@ -28,22 +29,22 @@ function PositionPanel( { selectedElements, onSetProperties } ) {
 	return (
 		<Panel onSubmit={ handleSubmit }>
 			<Title>
-				{ 'Position' }
+				{ __( 'Position', 'amp' ) }
 			</Title>
 			<InputGroup
-				label="X"
+				label={ __( 'X', 'amp' ) }
```
Comment (swissspidy) [inline assets/src/edit-story/panels/position.js]: Let's use `_x()` to explain what "X", "Y", and "px" mean here.

File: assets/src/edit-story/panels/size.js [hunk 4]
```diff
@@ -44,7 +45,7 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 						height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
 					} );
 				} }
-				postfix="px"
+				postfix={ __( 'px', 'amp' ) }
```
Comment (swissspidy) [inline assets/src/edit-story/panels/size.js]: Same here about using `_x()` for "px" in this file.

File: assets/src/edit-story/app/history/useHistoryReducer.js (hunk 1 above)
Comment (spacedmonkey) [reply assets/src/edit-story/app/history/useHistoryReducer.js]: Do you want me to remove this?

File: assets/src/edit-story/elements/index.js [hunk 5]
```diff
@@ -11,6 +14,10 @@ import * as imageElement from './image';
 import * as squareElement from './square';
 import * as videoElement from './video';
 
+/**
```
Comment (spacedmonkey) [inline assets/src/edit-story/elements/index.js]: I need to remove this.

File: assets/src/edit-story/panels/index.js (hunk 2 above)
Comment (spacedmonkey) [reply assets/src/edit-story/panels/index.js]: I will remove.

File: assets/src/edit-story/app/history/useHistoryReducer.js (hunk 1 above)
Comment (spacedmonkey) [reply assets/src/edit-story/app/history/useHistoryReducer.js]: Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/d4cd0a55ce8b81f0ad639c1f2c7a244444fafc34

File: assets/src/edit-story/panels/index.js (hunk 2 above)
Comment (spacedmonkey) [reply assets/src/edit-story/panels/index.js]: Removed in https://github.com/ampproject/amp-wp/pull/4143/commits/d4cd0a55ce8b81f0ad639c1f2c7a244444fafc34

File: assets/src/edit-story/panels/size.js (hunk 4 above)
Comment (spacedmonkey) [reply assets/src/edit-story/panels/size.js]: Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/8e29fd2233e19ebfeb7f1d1dfdda23ca3dc8fc19

File: assets/src/edit-story/panels/position.js (hunk 3 above)
Comment (spacedmonkey) [reply assets/src/edit-story/panels/position.js]: Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/8e29fd2233e19ebfeb7f1d1dfdda23ca3dc8fc19

**Comment by spacedmonkey** [comment]
@swissspidy I am going to leave the loading of translations to you in the new plugin.

**Pull request merged**